tools/system_tests.bat

```diff
@@ -0,0 +1,48 @@
+@echo off
+
+echo Testing nidcpower
+pushd generated\nidcpower
+c:\python38\python.exe -m tox -c tox-driver.ini
```

Comment:
Can we not hardcode the Python version and path?

Reply:
To do that would require making sure the version we want is the first version on the system path on the test machine. I would rather be explicit about it here than take a chance that the wrong version is used.